examples/caching_and_replay/README.md

```diff
@@ -0,0 +1,42 @@
+# Caching and Replaying Schelling
+
+## Summary
+
+This example applies caching on the Mesa [Schelling example](https://github.com/projectmesa/mesa-examples/tree/main/examples/Schelling).
+It enables a simulation run to be "cached" or in other words recorded. The recorded simulation run is persisted on the local file system and can be replayed at any later point.
+
+It uses the [Mesa-Replay](https://github.com/Logende/mesa-replay) library and puts the Schelling model inside a so-called `CachableModel` wrapper that we name `CachableSchelling`.
+From the outside perspective, the new model behaves the same way as the original Schelling model, but additionally supports caching.
```

Comment:
Should be "from the user's perspective".

Reply:
changed